eZ/Publish/Core/Search/Slot.php

```diff
@@ -1,13 +1,13 @@
 <?php
 /**
- * File containing the Solr\Slot abstract class
+ * This file is part of the eZ Publish Kernel package
  *
  * @copyright Copyright (C) eZ Systems AS. All rights reserved.
  * @license For full copyright and license information view LICENSE file distributed with this source code.
  * @version //autogentag//
  */
 
-namespace eZ\Publish\Core\Search\Solr;
+namespace eZ\Publish\Core\Search;
```

Comment:
Moved to `Core\Search\Common` in https://github.com/ezsystems/ezpublish-kernel/commit/3b5c3f3580be8311d951ad22cb34f4036c0426eb.